Week05/lesson10/homework/cash_register_starter_code/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Cash Register</title>
+  <link rel="stylesheet" href="css/style.css" media="screen" type="text/css" />
+</head>
+
+<body>
+  <div id="register">
+    <div id="ticket">
+      <h1>Thank You!</h1>
+      <table>
+        <tbody id="entries">
+        </tbody>
+        <tfoot>
+          <tr>
+            <th>Total</th>
+            <th id="total">$0.00</th>
+          </tr>
+        </tfoot>
+      </table>
+    </div>
+    <form id="entry">
+      <input id="newEntry" autofocus placeholder="How Much?">
+    </form>
+  </div>
+
```

Comment:
What happens if the user tries to input something that's not a number? You may need some HTML here (along with a nice `error` CSS class) to display some sort of message